kymatio/scattering1d/frontend/tensorflow_frontend.py

```diff
@@ -20,22 +31,46 @@ def __init__(self, J, shape, Q=1, T=None, max_order=2, oversampling=0,
 
 
 class TimeFrequencyScatteringTensorFlow(
-        ScatteringTensorFlow, TimeFrequencyScatteringBase):
-    def __init__(self, *, J, J_fr, shape, Q, T=None, oversampling=0, Q_fr=1,
-            F=None, oversampling_fr=0, out_type='array', backend='tensorflow', name='TimeFrequencyScattering'):
-        ScatteringTensorFlow.__init__(self,name=name)
-        TimeFrequencyScatteringBase.__init__(self, J=J, J_fr=J_fr, shape=shape,
-            Q=Q, T=T, oversampling=oversampling, Q_fr=Q_fr, F=F,
-            oversampling_fr=oversampling_fr, out_type=out_type, backend=backend)
+    ScatteringTensorFlow, TimeFrequencyScatteringBase
+):
+    def __init__(
+        self,
+        *,
+        J,
+        J_fr,
+        shape,
+        Q,
+        T=None,
+        oversampling=0,
+        Q_fr=1,
+        F=None,
+        oversampling_fr=0,
+        out_type="array",
+        backend="tensorflow",
+        format="joint",
```

Comment:
what does this do

Reply:
what does the format kwarg do

Reply:
the format kwarg organizes the output shape in 2D (`"time"`) or 3D (`"joint"`). in 2D, it returns `(path, time`) while 3D returns `(path, freq, time)`. the latter allows 2D convolution

Reply:
previously the kwarg was implemented but missing for the `torch` frontend. this PR resolves that

Reply:
can you link me to the code where it was implemented to do that stuff you mention?

Reply:
https://github.com/cyrusvahidi/kymatio/blob/6b35444be93e7cd9d19d8c5db061a5240d6bf239/kymatio/scattering1d/core/timefrequency_scattering.py#L409

Reply:
Thank you.